pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -99,11 +99,29 @@ export class ConfigurationCard extends React.Component {
         $("#system_information_ssh_keys").on("hide.bs.modal", () => this.host_keys_hide());
 
         this.realmd.addEventListener("changed", () => this.setState({}));
+
+        $(superuser).on("changed", () => {
+            if (superuser.allowed !== null)
+                this.systime_setup();
+        });
     }
 
     systime_setup() {
         const self = this;
 
+        if (self.server_time) {
+            self.server_time.close();
```

Comment:
What happens to the `poll_ntp_synchronized()` here? It looks like it would then try to do `client.call()` on a closed `client()`. How about doing something similar like commit c9751a178d7d084d6577 and do the reconnection inside serverTime.js?

Reply:
Yes, that was my first idea, but I think it turned out to be a biggish change. I'll try again with more brain. :-)

Reply:
Alright, the code in configurationCard and serverTime is pretty intertwined... I'll see how much effort it is to modernize this in general (ServerTime provides all the state, configurationCard does all the rendering in React.)

Reply:
Seems quite a big effort, there is lots of jQuery to clean up. Let's plan this properly.

Reply:
This code still looks wrong to me. After that, `self.server_time` is essentially kaputt, no? So it shouldn't receive new `.call()`s, nor should the timer below still execute.

Reply:
`self.server_time` gets re-initialized immediately again on line 124. All calls will use the new one from that point on, including the `poll_ntp_synchronized` call in the polling loop.

Reply:
Ah, I see it now. I must say, this is rather confusing..